WEB/BE/services/auth/index.js

```diff
@@ -0,0 +1,26 @@
+const model = require('@models/sequelizeWEB.js');
+const authsModel = model.auths;
+function authService() {}
```

Comment:
function 보다는 const를 이용한 객체로 선언해도 좋을 것 같습니다!
```suggestion
const authService = {
  check() { ... },
  insert() { ... },
}
```

Reply:
const authService = () =>{} 이렇게 하자고 말씀하신거 맞을까요 ???!!
그냥 객체로 하면 에러가 날 것 같아서 확인해보니 에러가 나네요 ..ㅎㅎ

Reply:
function보다 객체로 선언하면 개선되는 사항에는 어떤게 있을까요???

Reply:
음.. 큰 차이는 없을 것 같지만 보통 function은 하나의 함수 또는 생성자 함수를 만들 때 사용하는데 지금은 여러 키-값 쌍의 객체를 생성하는 목적이니 객체가 더 적합하다고 생각했습니다 ㅎㅎ